### PR TITLE
Add components data copy task

### PR DESCRIPTION
**Pull request by timomayer**

### CODE REVIEW

File: tasks/copy.js [hunk 1]
```diff
@@ -428,6 +428,25 @@ gulp.task('copy:dist:component:mock', function () {
   return copyFiles(from, to);
 });
 
+gulp.task('copy:dist:component:data', function () {
+  const from = path.join(
+    config.global.cwd,
+    config.global.src,
+    config.global.components,
+    '**',
+    'data',
```
Comment (dannystey) [inline tasks/copy.js]: how do we decide if a folder gets an underscore prefix or not?

Comment (SheepFromHeaven) [reply tasks/copy.js]: If you refer to a pattern that forbids building this resource...we don't have one. So _ files/folders will also be built.

**Comment by SheepFromHeaven** [comment]
Fixes #194 @timomayer next time if you do a ticket and a pr please link ;)

File: tasks/copy.js (hunk 1 above)
Comment (timomayer) [reply tasks/copy.js]: folders with _ are folders which are necessary for the dev environment/tests/styleguide but NOT used in live environments.

Comment (jurekbarth) [reply tasks/copy.js]: @timomayer true, but we're not handling them here right? @dannystey maybe you could open a new ticket for that, so we handle that for all folders :)

**Comment by jurekbarth** [comment]
@timomayer i've tested this feature and on build files are in `dist/components` and not `dist/resources/components` thats a bug right? Also we would need a `copy:dev` task in order to make it work in dev 👍

**Comment by timomayer** [comment]
you are right, Jurek

**Comment by smohadjer** [comment]
This ticket should be opened since as @jurekbarth said data folder is being copied to a wrong location.

**Comment by timomayer** [comment]
Now fixed with https://github.com/biotope/biotope-build/issues/200

**Pull request merged**